addons/purchase/models/account_invoice.py

```diff
@@ -142,6 +145,8 @@ def _anglo_saxon_purchase_move_lines(self, i_line, res):
             for line in res:
                 if line.get('invl_id', 0) == i_line.id and reference_account_id == line['account_id']:
                     valuation_price_unit = i_line.product_id.uom_id._compute_price(i_line.product_id.standard_price, i_line.uom_id)
+                    interim_account_price = float_round(valuation_price_unit * line['quantity'], precision_digits=account_prec)
```

Comment:
if there is a bug, please make a PR to the proper stable version where it ocured first, with use case explained and so on